### PR TITLE
Get form styled and working, minor tweaks in footer, remove unused fonts, swap matts pic

### PR DESCRIPTION
**Pull request by mjsz**

### CODE REVIEW

File: src/components/contact-form.js [hunk 1]
```diff
@@ -11,8 +11,11 @@ const FormSuccess = () => (
 
 
 const FormError = ({ errorMsg }) => (
-  <div className="EmailCaptureForm-error" dangerouslySetInnerHTML={{ __html: errorMsg }}>
-  </div>
+    <div className="ContactForm-error">
+        Oops! Something went wrong.
+        <div dangerouslySetInnerHTML={{ __html: errorMsg }}>
```
Comment (trotterdylan) [inline src/components/contact-form.js:16]: does this need to be html or is plain text sufficient?

File: src/components/contact-form.js [hunk 2]
```diff
@@ -61,33 +64,40 @@ const ContactForm = () => {
...
+                'form-name': form.getAttribute('name'),
             }),
-        }).then((response) => {
-            let res = response.clone()
-            let resJson = res.json()
-
-            if (res.status === 200 && resJson.success) {
+        }).then(response => {
+            if (!response.ok) {
+	        console.log('! form server response: ', response)
```
Comment (trotterdylan) [inline src/components/contact-form.js:82]: fix indentation (or maybe get rid of this?)

File: src/components/contact-form.js [hunk 3]
```diff
@@ -61,33 +64,40 @@ const ContactForm = () => {
...
+                throw new Error('HTTP error ' + response.status);
+            }
+            return response.json();
+        }).then(resJson => {
+            if (resJson.success) {
                 sendGaEvent()
                 setSuccess()
             }
 	    else {
```
Comment (trotterdylan) [inline src/components/contact-form.js:91]: indentation

File: src/components/contact-form.js (hunk 1 above)
Comment (mjsz) [reply src/components/contact-form.js:16]: it's sending a reference to our email address from the server with a <a mailto=""> so html

Comment (mjsz) [reply src/components/contact-form.js:16]: could rejigger to just send down the email, but <shrug>

Comment (mjsz) [reply src/components/contact-form.js:16]: we could rejigger to pass only the email down but whatev

**Pull request merged**